server/src/main.rs

```diff
@@ -1,3 +1,22 @@
-fn main() {
-    println!("Hello, world!");
+mod routes;
+
+use std::io;
+
+use actix_files::Files;
+use actix_web::{middleware::Logger, App, HttpServer};
+
+#[actix_web::main]
+async fn main() -> io::Result<()> {
+    env_logger::init();
+
+    HttpServer::new(|| {
+        App::new()
+            .wrap(Logger::default())
+            .service(routes::login::handler)
```

Comment:
This is fine for now -- we may want to move responsibility of configuring groups of routes (such as a group containing `login` and `logout`) elsewhere using `.configure()`

Reply:
Also if we want to move them into an `auth` scope, we can use `web::scope()`

Reply:
Agree with both of the above. I think routes with shared prefixes can even create a `Scope` within the module itself, and then the caller does not need to worry about any of the details.